zbus/src/lib.rs

```diff
@@ -41,8 +41,14 @@ pub use dbus_error::*;
 mod error;
 pub use error::*;
 
-mod address;
-pub use address::*;
+pub mod address;
+// re-export mostly for compatibility reasons
+#[cfg(any(
+    all(feature = "vsock", not(feature = "tokio")),
+    feature = "tokio-vsock"
+))]
+pub use address::VsockAddress;
+pub use address::{Address, TcpAddress, TcpAddressFamily};
```

Comment:
This is half of what I meant. :) 

* Names should be shortened where possible: `address::{Vsock, Tcp, TcpFamily}`.
* We provide type aliases for compatibility but deprecate them to discourage future use.

Reply:
Yes, that's also my aim, however! I think the type should still convey enough meaning on it's own though. Tcp or Vsock would be a pretty bad type name, it would be confusing and easily conflict with other related parts.. even in zbus code itself. TcpFamily might be annough, although AddressFamily could be more appropriate and general for ex. So for address:: I don't think we should rename much.

Reply:
I realized that perhaps I'm getting a bit too hardcore about modules here. Perhaps you're right and we should still have `Address` in the names etc. However, probably still best to do this separate and for all modules.

Reply:
I don't see much point in doing all modules at the same time. It's a bit easier for case by case instead. But it's easy to move this patch to a separate Draft PR if you prefer.

Reply:
> I don't see much point in doing all modules at the same time.

consistency.

Reply:
see #410